_includes/navbar.html

```diff
@@ -24,6 +24,7 @@
           <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">About <span class="caret"></span></a>
           <ul class="dropdown-menu">
             <li><a href="/about-OBO-Foundry.html">About the OBO Foundry</a></li>
+            <li><a href="https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/docs/COC.md">Code of Conduct</a></li>
```

Comment:
Link the proper webpage:
http://obofoundry.org/docs/COC.html

`/COC.html`

Reply:
fixed, thank you!